### PR TITLE
To Do list: interactive list

### PR DESCRIPTION
**Pull request by ZuhRa-HashiMi**
Use webpack to bundle JavaScript.
Learn how to use proper ES6 syntax.
Use ES6 modules to write modular JavaScript.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+          el.index = i;
+        });
+        this.saveTolocal();
+        this.renderTodo(list);
+      }
+
+      removeItem(item) {
+        const idItem = item.id;
+        this.removeFromLocal(idItem);
+      }
```
Comment (christianbiring1) [inline src/index.js:118]: - [ ] Also you may consider the remove the completed tasks when the user clicks on `clear all completed`. 

`Hint: You may consider grabbing this removeItem fx in an click event`

- [ ] Also, for a better UX you may consider fixing the path of your image.
image

File: README.md [hunk 2]
```diff
@@ -1 +1,34 @@
...
+
+## Built With
+
+- HTML, CSS and JavaScript
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
```
Comment (christianbiring1) [inline README.md]: - [ ] Kinldy make sure you provide descriptive and easy-to-understand instructions on how to run the project locally.

`hint: You may say: To clone the repos run in the terminal git clone <REPO_LINK>`
`To navigate over it : cd <DIRECTORY_NAME>`
`You may also consider not forgetting about the dependencies insatllation` :smile: 
`...`

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @ZuhRa-HashiMi , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
# STATUS: REQUEST CHANGES :recycle: 
soclose


## Highlights
- [ ] All linters checks are passing :heavy_check_mark: 
- [ ] Descriptive PR `title` and `summary`
- [ ] Good commit message


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @ZuhRa-HashiMi   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

- [ ] As requested by the previous reviewer, kindly make sure when we click on the `Clear all completed` button every checked task is deleted. :wink: 

   - ### YOUR WEBSITE NOW
    chrome-capture-2022-10-12 (4)

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [OPTIONAL] Kindly make sure the task `index` must always start at `1` not from `0`.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 3]
```diff
@@ -1 +1,82 @@
...
+
+**## Steps to run the project
+
+- git clone `git@github.com:ZuhRa-HashiMi/To_Do_list_interactive-.git`
+
+- cd `To_Do_list_interactive-`
+
+- npm install
+
+- npm start.**
```
Comment (ElsonOtake) [inline README.md:29]: - [OPTIONAL] I think it's better to fix this command to correct instructions about how to run your project. Your project will look more professional and create a better UX.

Screen Shot 2022-11-12 at 3 13 52 PM

**Review by ElsonOtake — APPROVED** [review]
# status: APPROVED :green_circle: :100: 

Hi @ZuhRa-HashiMi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 

Minions

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️
- PR has a good title and summary. ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. 👍 👍 

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**